scripts/managed-services/get-kafka-instance-state/metadata.yaml

```diff
@@ -0,0 +1,30 @@
+file: get-kafka-instance-state.sh
+name: managed-services/get-kafka-instance-state
+description: Print out the logs for the specified Kafka instance. `oc logs <pod-running-job> | tar -xzf -` will create a directory with all instance information.
```

Comment:
Won't this fails if any of the commands are not connected as expected?

Also, it's. Good question on how to transfer data between the command outside

I can think of saving a tarfile inside and then adding to the tooling around the script an option to pull a file outside

Reply:
The idea was to gather the artefacts (which will comprise kube and kafka artefacts) in an tar archive which willl follow a standard format to ease debugging.  The CEE user of the `ocm  backplane managedjob` command can then direct the job's log to file to get the tgz.

Is there a better way to pull a file from ocm backplane job?

```
ocm  backplane managedjob logs openshift-job-h55l9 > mykafkadebug.tgz
tar xvf mykafkadebug.tgz
```